rust/cmsis-pack/Cargo.toml

```diff
@@ -22,7 +22,7 @@ rustc-demangle = "0.1.16"
 serde = { version = "1.0.118", features = ["derive"] }
 serde_json = "1.0"
 tokio = { version = "1.0", features = ["macros", "rt"] }
-reqwest = { version = "0.11.0", default_features = false, features = ["rustls-tls", "trust-dns", "stream"] }
+reqwest = { version = "0.11.0", default_features = false, features = ["rustls-tls-native-certs", "trust-dns", "stream"] }
```

Comment:
I think the feature in `reqwest` is called `rustls-tls-native-roots` and uses the `rustls-tls-native-certs` crate.